analysis/descriptives/table2.R

```diff
@@ -124,7 +124,16 @@ print('Perform redaction')
 table2[,setdiff(colnames(table2),c("name","cohort","exposure","outcome","analysis"))] <- lapply(table2[,setdiff(colnames(table2),c("name","cohort","exposure","outcome","analysis"))],
                                                                                                 FUN=function(y){roundmid_any(as.numeric(y), to=threshold)})
 
+# Rename columns (output redaction) --------------------------------------------
+
+names(df)[names(df) == "unexposed_events"] <- 'unexposed_events_midpoint6'
+names(df)[names(df) == "exposed_events"] <- "exposed_events_midpoint6"
+names(df)[names(df) == "total_events"] <- "total_events_midpoint6"
```

Comment:
Fixed.

Reply:
Please can you direct me to the line number where this is fixed? As far as I can tell, total_events is still being midpoint6 rounded rather than derived from unexposed and exposed events.

Reply:
Line 131 following this link [column names](https://github.com/opensafely/post-covid-neurodegenerative/pull/72/commits/51ac8ee6d45b404eb9858f62ae407e6416f63a8e)

Reply:
Line 131 does not address this comment. You need to recalculate `total_events_midpoint6` by summing `unexposed_events_midpoint6` and `exposed_events_midpoint6` so the table adds up.

Reply:
Clarification added via email. Thanks.